test/performance/scenario4.py

```diff
@@ -0,0 +1,40 @@
+import json
+from locust import HttpUser, task, LoadTestShape, FastHttpUser, tag, constant
+import config as config
+
+
+class lastWaitTime:
+    def __init__(self):
+        self.wait_time = 0
+
+    def add_last_wait_time(self, add_wait_time: int):
+        self.wait_time += add_wait_time
+
+    def get_last_wait_time(self):
+        return self.wait_time
+
+
+last_wait_time = lastWaitTime()
+
+
+def increasing(wait_time):
```

Comment:
This seems to be a bit of an orphaned method? It only seems to be used inside the FiveHundredDest() class - could it not be a class method?